src/ipfsd-daemon.js

```diff
@@ -228,43 +247,55 @@ class Daemon {
   /**
    * Stop the daemon.
    *
+   * @param {object} options
+   * @param {number} options.timeout - How long to wait for the daemon to stop
```

Comment:
nit: specify as optional and include default
```suggestion
   * @param {object} [options]
   * @param {number} [options.timeout=60000] - How long to wait for the daemon to stop
```